E2_ESRGAN/config.yaml

```diff
@@ -1,7 +1,33 @@
-checkpoint_path: "checkpoints/"
+checkpoint_path:
+  phase_1: "checkpoints/phase_1"
+  phase_2: "checkpoints/phase_2"
 RDB:
   residual_scale_beta: 0.2
-adam:
-  initial_lr: 0.0001
-  beta_1: 0.9
-  beta_2: 0.999
+dataset: 
+  name: "open_images_v4"
+  scale_method: "bicubic"
+  hr_dimension: 512
+#Training
+# warmup_num_iter: 1000 # Number of steps to train the PSNR model
+batch_size: 128
+iterations: 2
+train_psnr:
+  adam:
+    initial_lr: 0.0002
```

Comment:
Are these taken from the paper? If yes, add a comment saying so.

Reply:
Yep

Reply:
Sure